### PR TITLE
Improving readme with more detail

### PR DESCRIPTION
**Pull request by clone1018**
Added more detail to our readme, including steps on development & testing!

@mja00 @haydenmc @EmberShadow Let me know if this still makes sense for all of your use cases!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,27 +47,36 @@ You will need the following tools to clone, build, and run Glimesh:
...
-
-## Cloning
+### Cloning
 
 To clone a local copy of Glimesh.tv, run
 
 ```sh
-git clone git@github.com:Glimesh/glimesh.tv.git
+git clone https://github.com/Glimesh/glimesh.tv.git
+# or if you have permissions to the repository
```
Comment (wolfcomp) [inline README.md:58]: i would say change this to say `or if you want to use ssh key auth to git for the repo` as the ssh path is only if you are logged in to git on a global level and not repo level

File: README.md [hunk 2]
```diff
@@ -27,27 +47,36 @@ You will need the following tools to clone, build, and run Glimesh:
...
 **NOTE**: If you are using WSL2, ensure you are cloning inside of your WSL2 instance
 (ex. `/home/user/...`) and not inside of a mounted Windows drive (ex. `/mnt/c/Users/...`)
 as this can have a significant negative impact on performance.
 
-## Configuring Postgres
+### Windows Installation
+
+Running Glimesh.tv natively on Windows is not yet understood. However you can run the application very well with WSL2. If you are interested in running Glimesh.tv natively, have a go at us and let us know!
+
+[WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with Ubuntu 20.04 has been tested successfully for development on Windows.
```
Comment (wolfcomp) [inline README.md:70]: i have tested this with the 18.04 version aswell as that is what im running on and it also works

Comment (clone1018) [reply README.md]: Updated, thanks!

File: README.md (hunk 1 above)
Comment (clone1018) [reply README.md]: Updated as well

**Review by haydenmc — APPROVED** [review]
Looks good to me!

**Pull request merged**